src/controls/Controls.spec.js

```diff
@@ -1 +1,11 @@
-// Test away!
\ No newline at end of file
+// Test away! 
+import React from "react";
+import { render, fireEvent } from "react-testing-library";
+import Controls from "./Controls";  
+ 
+describe("<Controls />", () => {
+    it("Default state set to unlocked and open", () => {
+      const { getByText } = render(<Controls />);
+    }) 
+})  
+ 
```

Comment:
Okay, so these are the MVP requirements for the Controls component:

    provide buttons to toggle the closed and locked states.
    buttons' text changes to reflect the state the door will be in if clicked
    the closed toggle button is disabled if the gate is locked
    the locked toggle button is disabled if the gate is open

I'm not seeing these (besides the second one) in Controls or any of your other tests. You'll need to do these.